### PR TITLE
Additional Coding Schemes

### PR DESCRIPTION
**Pull request by IsaackMwenda**
@lukechurch Kindly review

### CODE REVIEW

File: coding_schemes/Needs_Met_Yesno.json [hunk 1]
```diff
@@ -98,6 +98,22 @@
...
+      "CodeType": "Normal",
+      "StringValue": "Rent",
+      "DisplayText": "Rent"
+    },
+    {
+      "VisibleInCoda": true,
+      "NumericValue": 13,
+      "CodeID": "code-0bf74d7f6cbf",
+      "CodeType": "Normal",
+      "StringValue": "Clothes",
```
Comment (lukechurch) [inline coding_schemes/Needs_Met_Yesno.json]: To be consistent with the others, the string values should be lower case

File: coding_schemes/Cash_Modality_Yesno.json [hunk 2]
```diff
@@ -89,14 +89,6 @@
       "CodeID": "code-564016a9184b",
       "CodeType": "Normal"
     },
-    {
```
Comment (lukechurch) [inline coding_schemes/Cash_Modality_Yesno.json]: We can only do this if there is no data coded with this code - I've just confirmed this, please please can you explicitly state it on future PRs that delete codes?

File: coding_schemes/Needs_Met_Yesno.json (hunk 1 above)
Comment (IsaackMwenda) [reply coding_schemes/Needs_Met_Yesno.json]: Noted

File: coding_schemes/Cash_Modality_Yesno.json (hunk 2 above)
Comment (IsaackMwenda) [reply coding_schemes/Cash_Modality_Yesno.json]: Thank you @lukechurch . This is noted.

**Pull request merged**